setup.py

```diff
@@ -164,7 +164,7 @@ def additional_commands():
             Topic :: Software Development :: Libraries
             """.splitlines() if len(c.strip()) > 0],
         description='traitsui: traits-capable user interfaces',
-        long_description=open('README.rst').read(),
+        long_description=open("README.rst").read().split(".. end_of_long_description")[0],
```

Comment:
Same orthogonal comment as in https://github.com/enthought/pyface/pull/617#discussion_r458616860
(Commenting here too because I can't assume which one gets read first)
(Please feel free to ignore and merge away.)

Reply:
I will merge now and then take a look at changing this later today. Thank you!